.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '16' # Update this to match your project's Node version
```

Comment:
node 16 is verrry old. consider newer version if possible

Reply:
That file was actually not added.